src/components/Repository.js

```diff
@@ -96,6 +97,15 @@ function Repository({match}) {
                 {repository && repository.languages.totalCount > languagesShown && `+${repository.languages.totalCount - languagesShown} languages`}
               </span>
             </div>
+            <Flex>
+              {repository && (
+                <a rel="noreferrer" target="_blank" href={`https://codetriage.com/${nameWithOwner}`}>
+                  <Button primary>
+                    Watch on Codetriage
```

Comment:
I think it'll be a good idea to follow the Casing convention of their brand name, ie. CodeTriage and not Codetriage.

![image](https://user-images.githubusercontent.com/1915009/85037896-940eb980-b1a3-11ea-9ae4-9f6e2e8fcd44.png)

Reply:
@sujitagarwal Good catch! I will update it! :+1: